src/components/Locations.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react'
+import axios from "axios";
+
+class Locations extends Component {
+    constructor() {
+        super();
+        this.state = {
+            locations: [],
+            displayList: false,
+        }
+    }
+
+    grabLocations = async () => {
+        const { data } = await axios.get("https://ghibliapi.herokuapp.com/locations/");
+
+        this.setState ({
+            locations: data,
+        })
+    }
+
+    handleClick = () => {
+        this.setState({
+            displayList: !this.state.displayList,
+        })
+    }
+
+    componentDidMount() {
+        this.grabLocations();
+    }
+
+    render() {
+        const { locations, displayList } = this.state;
+        const locate = locations.map((location, i) => <li key={i}>
```

Comment:
I don't think `locate` is a great name for this variable. (We want only GREAT names here at GhibliApp!) It's easy to mislead the code reader (likely yourself!), since it sounds like a function/verb. You aren't printing "locate" below... You're printing multiple... somethings. `locationListItems`? `locationItems`? `displayedLocations`? None of those are great either, but it's harder to figure out what your code does if your names are misleading. Awkward is better than misleading!

Reply:
That makes sense, I'll work on my naming convetions.